web/src/router/index.js

```diff
@@ -1,12 +1,15 @@
 import {createRouter, createWebHashHistory} from 'vue-router';
 
+import DsProtMain from '../components/dsprot/DsProtMain.vue'
```

Comment:
```suggestion
const DsProtMain = () => import("@/components/dsprot/DsProtMain.vue")
```

I think it's nicer to stick to lazy loading routes :)

Reply:
Seems to work, changed in next commit. What exactly is the benefit here?